iroh-car/README.md

```diff
@@ -1,9 +1,15 @@
 # iroh-car
 
-> [CAR file](https://ipld.io/specs/transport/car/) support for iroh.
+[CAR file](https://ipld.io/specs/transport/car/) support for iroh. "CAR" stands
+for Content Addressable aRchives. A CAR file typically contains a serialized
+representation of an [IPLD
+DAG](https://docs.ipfs.tech/concepts/merkle-dag/#merkle-directed-acyclic-graphs-dags),
+though is general enough to citnain arbitrary IPLD blocks.
```

Comment:
"cintain" -> "contain"

Reply:
Sorry, pressed the wrong button to rerequest review, will fix soon.

Reply:
Fixed for real now!